### PR TITLE
feat: feature lifecycle complete and uncomplete

### PR DESCRIPTION
**Pull request by sjaanus**
Creating a way to complete and uncomplete feature.

### CODE REVIEW

File: src/lib/features/feature-lifecycle/feature-lifecycle-service.ts [hunk 1]
```diff
@@ -79,11 +90,6 @@ export class FeatureLifecycleService extends EventEmitter {
                 ),
             );
         });
-        this.eventStore.on(FEATURE_COMPLETED, async (event) => {
```
Comment (sjaanus) [inline src/lib/features/feature-lifecycle/feature-lifecycle-service.ts:82]: This is not needed, this event is not coming from anything external.

File: src/lib/features/feature-lifecycle/feature-lifecycle-controller.ts [hunk 2]
```diff
@@ -62,6 +65,46 @@ export default class FeatureLifecycleController extends Controller {
...
+                        ...getStandardResponses(401, 403, 404),
+                    },
+                }),
+            ],
+        });
+
+        this.route({
+            method: 'post',
+            path: `${PATH}/uncomplete`,
+            handler: this.unComplete,
```
Comment (kwasniew) [inline src/lib/features/feature-lifecycle/feature-lifecycle-controller.ts:92]: uncomplete

File: src/lib/features/feature-lifecycle/feature-lifecycle-service.ts [hunk 3]
```diff
@@ -139,14 +145,32 @@ export class FeatureLifecycleService extends EventEmitter {
...
+        );
+    }
+
+    public async featureUnCompleted(feature: string, auditUser: IAuditUser) {
+        await this.featureLifecycleStore.deleteStage({
+            feature,
+            stage: 'completed',
+        });
+        await this.eventService.storeEvent(
+            new FeatureUnCompletedEvent({
```
Comment (kwasniew) [inline src/lib/features/feature-lifecycle/feature-lifecycle-service.ts:169]: FeatureUncompletedEvent

File: src/lib/features/feature-lifecycle/feature-lifecycle.e2e.test.ts [hunk 4]
```diff
@@ -60,6 +60,21 @@ const getFeatureLifecycle = async (featureName: string, expectedCode = 200) => {
...
 };
+const completeFeature = async (featureName: string, expectedCode = 200) => {
+    return app.request
+        .post(
+            `/api/admin/projects/default/features/${featureName}/lifecycle/complete`,
+        )
+        .expect(expectedCode);
+};
+
+const unCompleteFeature = async (featureName: string, expectedCode = 200) => {
```
Comment (kwasniew) [inline src/lib/features/feature-lifecycle/feature-lifecycle.e2e.test.ts:71]: uncompleteFeature

**Pull request merged**